product_docs/docs/pgd/5/parallelapply.mdx

```diff
@@ -38,17 +48,23 @@ WHERE sub_name = 'bdr_bdrdb_bdrgroup_node2_node1';
 SELECT bdr.alter_subscription_enable ('bdr_bdrdb_bdrgroup_node2_node1');
 ```
 
-### When to use Parallel Apply
+### When to use Parallel apply
+
+Parallel apply is always on by default and for most operations, we recommend that it is left on.
+
+### When not to use Parallel apply
+
+From PGD 5.2, Parallel apply works with Group Commit, CAMO and eager replication. 
 
-Parallel Apply is always on by default and for most operations, we recommend that it is left on.
+### Monitoring Parallel apply
 
-### When not to use Parallel Apply
+To support Parallel apply's deadlock mitigation, PGD 5.2 adds columns to [`bdr.stat_subscription`](/pgd/latest/reference/catalogs-visible#bdrstat_subscription). The new columns are `nprovisional_waits`, `ntuple_waits` and `ncommmit_waits`. The `nprovisional_waits` value reflects the number of operations on the same tuples being performed by concurrent apply transactions. These are provisional waits which are not actually waiting yet but could. 
 
-As of, and up to at least PGD 5.1, Parallel Apply should not be used with Group Commit, CAMO and eager replication. You should disable Parallel Apply in these scenarios. If you are experiencing a large number of deadlocks, you may also want to disable it.
+If apply operations are affected by the deadlock mitigation measures, they will wait until they can be safely applied and are counted in `ntuple_waits`. Fully applied transactions which waited before being committed are counted in `ncommit_waits`. 
```

Comment:
```suggestion
If apply operations are affected by the deadlock mitigation measures, they wait until they can be safely applied and are counted in `ntuple_waits`. Fully applied transactions which waited before being committed are counted in `ncommit_waits`. 
```

Reply:
I think the "will" is appropriate as, as I understand it they won't be waiting quite yet (there's some prediction going on) but they will wait when then time comes for them to be applied.